### ISSUE TITLE
Export to csv or xls  when an area is given not exporting area data for all range values

### ISSUE THREAD

**Issue by aykutyararbas** [highslide-software/export-csv#42]
var chart = new Highcharts.Chart({
				   chart: {
					   renderTo: 'container',
			           zoomType: 'xy' 
			      },
		        title: {
		            text: metricId
		        },

		        xAxis: {
		            type: 'datetime'
		        },

		        yAxis: {
		            title: {
		               align: 'high',
		                offset: 0,
		                text: '%',
		                rotation: 0,
		                y: 6
		            }
		        },

		        tooltip: {
		            crosshairs: true,
		            shared: true,
		            valueSuffix: '%'
		        },

		         legend: {
		            layout: 'horizontal',
		            align: 'left',
		            x: 50,
		            verticalAlign: 'top',
		            y: 20,
		            floating: true ,
		            itemStyle: {
		             "color": "#333333", 
		             "cursor": "pointer", 
		             "fontSize": "10px", 
		             "fontWeight": "regular" 
		            }
		        },
		        series: [{
		            name: 'CPU Utilization',
		            data: averages,
		            zIndex: 1,
		            marker: {
		                fillColor: 'white',
		                lineWidth: 2,
		                lineColor: Highcharts.getOptions().colors[0]
		            }
		        }, {
		            name: 'Band',
		            data: ranges,
		            type: 'arearange',
		            lineWidth: 0,
		            linkedTo: ':previous',
		            color: Highcharts.getOptions().colors[0],
		            fillOpacity: 0.3,
		            zIndex: 0
		        }]
		    });

**Comment by highslide-software** [comment]
Can you set up a live sample on jsFiddle? Your sample is not reproduceable because the `averages` and `ranges` variables are undefined.

**Comment by aykutyararbas** [comment]
Thanks for the quick response. 
Here is a fiddle example ;

http://jsfiddle.net/aykutyararbas/d4c26mrf/

Please let me know if you can access or not .

Thanks,

**Comment by highslide-software** [comment]
Yes I see it. It applies to `arearange` series. The same probably applies to all series types with more than one data value, like bubbles, heat maps, ohlc and candlestick.
